src/dcd/server/autocomplete/util.d

```diff
@@ -763,7 +763,16 @@ AutocompleteResponse.Completion makeSymbolCompletionInfo(const DSymbol* symbol,
 {
 	string definition;
 	if ((kind == CompletionKind.variableName || kind == CompletionKind.memberVariableName) && symbol.type)
-		definition = symbol.type.name ~ ' ' ~ symbol.name;
+	{
+		if (symbol.type.kind == CompletionKind.functionName && symbol.type.type)
+		{
+			string retTypeName = symbol.type.type.name;
+			string fnName = symbol.type.name;
+			definition = fnName ~ "()->" ~ retTypeName ~ " " ~ symbol.name;
```

Comment:
this looks like a C++ method on a pointer, can you add spaces or use a different separator instead?

Reply:
so can you add spaces around the `->`?

```suggestion
			definition = fnName ~ "() -> " ~ retTypeName ~ " " ~ symbol.name;
```

Reply:
can you add a test with extended output? I thought serve-d would just take what is printed there as-is